src/Symfony/Component/DependencyInjection/Definition.php

```diff
@@ -30,6 +30,8 @@ class Definition
     private $factoryMethod;
     private $factoryService;
     private $shared = true;
+    private $deprecated = false;
+    private $deprecationTemplate = 'The service %service_id% is deprecated. You should stop using it, as it will soon be removed.';
```

Comment:
To be a bit more consistent with the existing deprecation messages, we could reword this a bit:

```php
"The service %service_id% is deprecated. You should stop using it, as it will soon be removed."

"The '%service_id%' service is deprecated. You should stop using it, as it will soon be removed."
````